LyokoAPI/Events/VirtualWorldDestructionEvent.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Reflection;
+using LyokoAPI.API;
+using LyokoAPI.VirtualStructures;
+using LyokoAPI.VirtualStructures.Interfaces;
+
+namespace LyokoAPI.Events
+{
+    class VirtualWorldDestructionEvent
+    {
+        private static event Events.OnVirtualWorldEvent VirtualWorldE;
+        public static void Call(IVirtualWorld world)
```

Comment:
There should also be a way of calling this with a string. See the virtualization events for examples.